types.go

```diff
@@ -28,6 +28,7 @@ type Request struct {
 	Payload     []byte      `protobuf:"bytes,3,opt,name=payload,proto3"`
 	TimeoutNano int64       `protobuf:"varint,4,opt,name=timeout_nano,proto3"`
 	Metadata    []*KeyValue `protobuf:"bytes,5,rep,name=metadata,proto3"`
+	// Files       *FileList   `protobuf:"bytes,6,rep,name=files,proto3"`
```

Comment:
Is this commented out accidentally or intentionally?

Reply:
Intentional, not needed and should be removed.

Reply:
Fixed this.